stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -569,25 +569,25 @@ spec:
         emptyDir:
           sizeLimit: {{ .Values.artifactory.persistence.size }}
   {{- end }}
+{{- with .Values.artifactory.persistence }}
+  {{- if and .enabled (not .existingClaim) }}
   volumeClaimTemplates:
-  {{- if .Values.artifactory.persistence.enabled }}
   - metadata:
       name: artifactory-volume
-    {{- if not .Values.artifactory.persistence.existingClaim }}
     spec:
-      {{- if .Values.artifactory.persistence.storageClassName }}
```

Comment:
Should this be `storageClasaName` or `storageClass`? because this could break compatibility for users